### PR TITLE
Fixing and Enhancing all rados suites to run on both BM and RHOS-d pipeline

### PR DESCRIPTION
**Pull request by pdhiran**
Changes made to:
1. All rados conf & Suite files changed to have client node as node 7 on all types of deployments.
2. Updated tier-2-rados-basic-regression.yaml to delete pools post execution
3. Updated tier-2_rados_ec-pool_recovery.yaml to change the EC profiles being tested.
4. Updated tier-2_rados_test-ecpool-osd-rebalance.yaml & tier-2_rados_test-mon-db-trimming.yaml with new clients. No major modifications here.

### CODE REVIEW

**Comment by pdhiran** [comment]
Pass logs :
1. tier-2-rados-basic-regression.yaml :
7.x : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-4A53XJ/ 
6.x : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-S8FJXW 

2. tier-2_rados_ec-pool_recovery.yaml 
7.x: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-AA1ENU/ 
6.x : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-JLE9BW  

Note : The failure in compression tests is due to RHOS-d env. The test has passed in the Baremetal env, and similarly, it has been moved to BM pipeline suite. 
Passed in BM execution : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-ZE5ZFL/Compression_test_-_EC_pool_0.log 

3. tier-2_rados_test-ecpool-osd-rebalance.yaml
7.x : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-QVPG46/ 
6.x : 


BM Execution pass logs ( Tests were run on same cluster, without re-triggering cluster deployment ) :
1. tier-2-rados-basic-regression-bm.yaml : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-XB8L5E/ 
Note : Failure in heartbeat test is a intermittent issue, and the reason being host taking longer time to come up post testing. Timeouts to be increased. 

2. tier-2_rados_test-ecpool-osd-rebalance-bm.yaml : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-PMU8HU/ 
3. tier-2_rados_test-mon-db-trimming-bm.yaml : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-XNGZML/
4. tier-2_rados_test-osd-inprogress-rebalance-bm.yaml : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-Y8X8K6/ 
5. tier-2_rados_test-pool-functionalities-bm.yaml : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-17C0OI/

File: tests/rados/test_osd_heartbeat.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ def run(ceph_cluster, **kw):
     cephadm = CephAdmin(cluster=ceph_cluster, **config)
     rados_obj = RadosOrchestrator(node=cephadm)
     timeout = config.get("timeout", 150)
-    recovery_timeout = config.get("recovery_timeout", 500)
+    recovery_timeout = config.get("recovery_timeout", 2400)
```
Comment (harshkumarRH) [inline tests/rados/test_osd_heartbeat.py:29]: 40 mins is too big of a timeout to let a machine restart and come back online, 10/15 mins should be more than enough IMO

Comment (pdhiran) [reply tests/rados/test_osd_heartbeat.py:29]: I believe it's safe to add this, as the wait is not a blind sleep, and the status is checked periodically until the timeout. So I think it's fine.

File: tests/rados/test_bluestoretool_workflows.py [hunk 2]
```diff
@@ -50,7 +50,7 @@ def run(ceph_cluster, **kw):
 
     try:
         # Execute ceph-bluestore-tool --help
-        osd_id = random.randrange(len(osd_list))
+        osd_id = random.choice(osd_list)
```
Comment (SrinivasaBharath) [inline tests/rados/test_bluestoretool_workflows.py:53]: Any reason for changing from randrange to choice?

File: suites/reef/rados/tier-2_rados_test-osd-inprogress-rebalance.yaml [hunk 3]
```diff
@@ -78,7 +78,7 @@ tests:
       config:
         command: add
         id: client.1                      # client Id (<type>.<Id>)
-        node: node8                       # client node
+        node: node7                       # client node
         install_packages:
           - ceph-common
           - ceph-base
```
Comment (SrinivasaBharath) [inline suites/reef/rados/tier-2_rados_test-osd-inprogress-rebalance.yaml:84]: ceph-base is not required for client node. Can you remove the ceph-base.

File: tests/rados/test_bluestoretool_workflows.py (hunk 2 above)
Comment (pdhiran) [reply tests/rados/test_bluestoretool_workflows.py:53]: Yes, We need to select OSDs that are part of cluster, not just randomly select OSDs in a range. 

Failure log : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-LVA8L4/ceph-bluestore-tool_utility_0.log

File: suites/reef/rados/tier-2_rados_test-osd-inprogress-rebalance.yaml (hunk 3 above)
Comment (pdhiran) [reply suites/reef/rados/tier-2_rados_test-osd-inprogress-rebalance.yaml:84]: Will remove the unnecessary packages when i work on fixing the suite issues. for now, I have just changed the node for client deployment.

**Review by harshkumarRH — APPROVED** [review]
LGTM

File: suites/reef/rados/tier-2_rados_test-osd-inprogress-rebalance.yaml (hunk 3 above)
Comment (SrinivasaBharath) [reply suites/reef/rados/tier-2_rados_test-osd-inprogress-rebalance.yaml:84]: Sure Pawan.

**Comment by SrinivasaBharath** [comment]
LGTM.

**Review by SrinivasaBharath — APPROVED** [review]

**Review by neha-gangadhar — APPROVED** [review]
looks good

**Pull request merged**